### PR TITLE
Use new packager URL in e2e test

### PR DESCRIPTION
**Pull request by javache**

### CODE REVIEW

File: local-cli/generator-ios/templates/app/AppDelegate.m [hunk 1]
```diff
@@ -31,7 +31,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
    * on the same Wi-Fi network.
    */
 
-  jsCodeLocation = [NSURL URLWithString:@"http://localhost:8081/index.ios.bundle"];
+  jsCodeLocation = [NSURL URLWithString:@"http://localhost:8081/index.ios.bundle?platform=ios&dev=true"];
```
Comment (frantic) [inline local-cli/generator-ios/templates/app/AppDelegate.m]: Why is this needed? I thought `dev` is `true` by default, and `platform` can be inferred from `.ios` extension. @amasad ?

Comment (paramaggarwal) [inline local-cli/generator-ios/templates/app/AppDelegate.m]: Yes, I'm curious too.

Comment (ide) [inline local-cli/generator-ios/templates/app/AppDelegate.m]: Shouldn't this actually be `index.bundle?platform=ios`? Just like how you can write `require('button')` and the system automatically loads button.ios.js, it seems consistent for `index.bundle` to automatically refer to `index.ios.js`.
